### PR TITLE
Intel WDL updates 

### PR DESCRIPTION
**Pull request by Kmannth**
In our latest benchmarking efforts at Intel we found some new cpu and memroy values to use for Latency and Throuput settings.  There are some tool updates and some minor readme updates.

### CODE REVIEW

**Comment by bshifaw** [comment]
Hi @Kmannth, 
You may need to contact a member from the Intel group to get this PR approved.

**Comment by rpomaris** [comment]
@droazen @lbergelson - Do you know who might be the right Broadie to review this?  We have a proposed PR for the Intel /gatk-workflows repo

**Comment by droazen** [comment]
@mepowers I'd suggest a joint review with someone from my team to review the cpu/memory settings (you can assign me for now), plus @bshifaw to review the overall WDL changes. It would help facilitate the review process if you could provide a brief summary of the important changes that made a difference in performance.

File: PairedSingleSampleWf_noqc_nocram_optimized.wdl [hunk 1]
```diff
@@ -168,8 +171,9 @@ task SamToFastqAndBwaMemAndMba {
         PROGRAM_GROUP_NAME="bwamem" \
         UNMAPPED_READ_STRATEGY=COPY_TO_TAG \
         ALIGNER_PROPER_PAIR_FLAGS=true \
-        UNMAP_CONTAMINANT_READS=true \
-        ADD_PG_TAG_TO_READS=false
+        UNMAP_CONTAMINANT_READS=true
+        #UNMAP_CONTAMINANT_READS=true \
```
Comment (bshifaw) [inline PairedSingleSampleWf_noqc_nocram_optimized.wdl:175]: Why were these parameters commented out? In latest version of this workflow these tools still uses the parameters [link](https://github.com/gatk-workflows/gatk4-genome-processing-pipeline/blob/d78f3886fde1a7146f0318552cb2a335eac70dde/tasks/Alignment.wdl#L106).

File: PairedSingleSampleWf_noqc_nocram_optimized.wdl [hunk 2]
```diff
@@ -476,8 +482,8 @@ task MarkDuplicates {
       ${"READ_NAME_REGEX=" + read_name_regex} \
       OPTICAL_DUPLICATE_PIXEL_DISTANCE=2500 \
       ASSUME_SORT_ORDER="queryname" \
-      CLEAR_DT="false" \
-      ADD_PG_TAG_TO_READS=false
+      #CLEAR_DT="false" \
```
Comment (bshifaw) [inline PairedSingleSampleWf_noqc_nocram_optimized.wdl:485]: Why were these parameters commented out? In latest version of this workflow these tools still uses the parameters [link](https://github.com/gatk-workflows/gatk4-genome-processing-pipeline/blob/d78f3886fde1a7146f0318552cb2a335eac70dde/tasks/BamProcessing.wdl#L134).

**Review by bshifaw — COMMENTED** [review]
Looks fine in general. I had questions about some parameters that were commented out. They seem to be still in use in Broads version of the pipeline [here](https://github.com/gatk-workflows/gatk4-genome-processing-pipeline/tree/d78f3886fde1a7146f0318552cb2a335eac70dde). See comments for specific lines.

File: PairedSingleSampleWf_noqc_nocram_optimized.wdl (hunk 1 above)
Comment (Kmannth) [reply PairedSingleSampleWf_noqc_nocram_optimized.wdl:175]: These changes were something we inherited.  We can revert and test again.

File: PairedSingleSampleWf_noqc_nocram_optimized.wdl (hunk 2 above)
Comment (Kmannth) [reply PairedSingleSampleWf_noqc_nocram_optimized.wdl:485]: These changes were something we inherited. We can revert and test again.

Comment (bshifaw) [reply PairedSingleSampleWf_noqc_nocram_optimized.wdl:485]: A person from Davids team can better determine whether these parameters are necessary, I just wanted to point out the difference and provide a link to the Broads current pipeline.

File: PairedSingleSampleWf_noqc_nocram_optimized.wdl (hunk 1 above)
Comment (bshifaw) [reply PairedSingleSampleWf_noqc_nocram_optimized.wdl:175]: A person from Davids team can better determine whether these parameters are necessary, I just wanted to point out the difference and provide a link to the Broads current pipeline.

**Comment by rpomaris** [comment]
@bshifaw @droazen -- @Kmannth is re-running our internal throughput testing with the two changes reverted.  He's been working on this the last week.  Tests are running and in progress.

**Comment by Kmannth** [comment]
Just as a quick update. We made these chages to support running on 192GB system  (and they work fine with our test workkload) but now we tend ask users to have  368GB.  Testing shows that the orignal memory values will fit and perform well on the larger memory system.  

I will update  with a new pull request witht the orignal memory values and have version for both hyper threaded and non hyperthreaded systems this week.

**Comment by Kmannth** [comment]
I will create a new pull request of the version 2.    The min memory values are now based on the Existing values from the 2T and 56T JSON WGS files.

**Pull request closed without merging**